introduction.md

```diff
@@ -3,4 +3,9 @@
 +
 +My favorite Code School paths are: 
 +*JavaScript
-+*HTML & CSS
\ No newline at end of file
++*HTML & CSS
++
++My deadly skills are:
++*JavaScript
++*HTML & CSS
++*Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?